pom.xml

```diff
@@ -135,6 +140,21 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>org.jboss.narayana.jta</groupId>
+        <artifactId>narayana-jta</artifactId>
+        <version>5.6.3.Final-SNAPSHOT</version>
+      </dependency>
+      <dependency>
+        <groupId>org.jboss.narayana.jta</groupId>
+        <artifactId>tomcat-jta</artifactId>
+        <version>5.6.3.Final-SNAPSHOT</version>
+      </dependency>
```

Comment:
IMO this is the only thing we need now - once you can confirm I will do the Narayana release and roll this to non-SNAPSHOT version